Shared/Flows/Onboarding/OnboardingCoordinator.swift

```diff
@@ -135,8 +135,8 @@ extension OnboardingCoordinator: OnboardingViewControllerDelegate {
                 try await user.saveLocalThenServer()
 
                 self.goToNextContentOrFinish()
-            } catch  {
-                logError(error)
+            } catch {
+                await ToastScheduler.shared.schedule(toastType: .error(error))
```

Comment:
I attempted to add error toast wherever there was user behavior that failed. If you find any other appropriate places, feel free to add.